javascripts/background_scripts/reading_process.js

```diff
@@ -140,55 +36,36 @@ var readingProcess = {
     var pattern = /privlyInjectableApplication\=/i;
     url = url.replace(pattern, "privlyApp=");
 
-    if( url.indexOf("privlyApp=Message") > 0 ) {
+    if (url.indexOf("privlyApp=Message") > 0) {
       path = "privly-applications/Message/show.html?privlyOriginalURL=";
-    } else if( url.indexOf("privlyApp=ZeroBin") > 0) {
+    } else if (url.indexOf("privlyApp=ZeroBin") > 0) {
       path = "privly-applications/Message/show.html?privlyOriginalURL="; // Deprecated
-    } else if( url.indexOf("privlyApp=PlainPost") > 0) {
+    } else if (url.indexOf("privlyApp=PlainPost") > 0) {
       path = "privly-applications/PlainPost/show.html?privlyOriginalURL=";
-    } else if( url.indexOf("https://priv.ly") === 0 ) {
```

Comment:
Try to keep whitespace changes in their own pull requests. It makes it easier to focus attention where it is needed.